docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -19,7 +19,7 @@ This document is part of a collection on porting. The complete list of porting g
 Playbook
 ========
 
-No notable changes
+ * ``hash_behaviour`` now affects inventory sources. If you have it set to ``merge``, the data you get from inventory might change and you will have to update playbooks accordingly. If you're using the default setting (``overwrite``), you will see no changes. Inventory was ignoring this setting.
```

Comment:
Looks good. Should this be an inventory specific setting to avoid breaking things?

Reply:
Unsure this is 'long standing bug' IMHO, but I'm willing to entertain a toggle if needed, I doubt this affects many people (but those are famous last words).